### ISSUE TITLE
Remove remnant schema files

### ISSUE THREAD

**Issue by hrshdhgd** [mapping-commons/sssom-py#290]
The use of [these files](https://github.com/mapping-commons/sssom-py/tree/master/schema) have diminished after implementing `sssom-schema`.

**Comment by matentzn** [comment]
Didnt you fix this?

**Comment by hrshdhgd** [comment]
Yes, but the "Note" is why I didn't close it. I wasn't sure if you want this to be open or no.

**Comment by matentzn** [comment]
Nah its fine, no worries. cliquesummary also still there. Thank you!
